FrameworklessServer/Data/Services/UsersService.cs

```diff
@@ -32,9 +32,9 @@ public void Add(User user)
         
         public void Delete(string name)
         {
-            if (name == _owner.Name)
+            if (name == _owner.Name)  //todo null and not existing
                 throw new ArgumentException("Cindy owns this world! Cannot delete");
-
+            
             var allUsers = GetAllUsers();
```

Comment:
Will need to check if user exists in service to delete. If not, what should we do?